packages/timeline-state-resolver-types/src/index.ts

```diff
@@ -172,3 +172,10 @@ export enum ActionExecutionResultCode {
 	Error = 'ERROR',
 	Ok = 'OK',
 }
+
+export interface StringInterpolation {
+	key: string
+	args?: {
+		[k: string]: any
+	}
+}
```

Comment:
Slightly confused by the naming, perhaps rename to `TemplateString`? This could also add some documentation?

```suggestion
/** This resolves to a string, where parts can be defined by the datastore */
export interface TemplateString{
	/** The string template. Example: "http://google.com?q={{searchString}}" */
	key: string
	/** Values for the arguments in the key string. Example: { searchString: "TSR" } */
	args?: {
		[k: string]: any
	}
}
```